code/modules/mob/living/carbon/human/species/station/station_vr.dm

```diff
@@ -1,3 +1,8 @@
+/datum/species/human
+	color_mult = 1
+	icobase = 'icons/mob/human_races/r_human_vr.dmi'
+	deform = 'icons/mob/human_races/r_def_human_vr.dmi'
```

Comment:
These three lines could probably be included at the bottom of this file. species/human is already used down there to define the humans egg type.